app/controllers/admin/users_controller.rb

```diff
@@ -34,8 +30,20 @@ def destroy
 
     private
 
+    def sort_column
```

Comment:
Do you think we can use this method in another controller as well?
So we can move it to concern. Thoughts?